code/game/objects/items/miscellaneous.dm

```diff
@@ -376,3 +376,67 @@
 	name = "living lube delivery beacon"
 	default_name = "Offensive"
 	mob_choice = /mob/living/simple_animal/hostile/retaliate/clown/lube
+
+//	-----------------
+//	Detective loadout
+//	-----------------
+
+/obj/item/choice_beacon/detective
+	name = "detective beacon"
+	desc = "There are many ways of investigating crime, pick yours."
+
+/obj/item/choice_beacon/detective/generate_display_names()
+	var/static/list/detective_item_list
+	if(!detective_item_list)
+		detective_item_list = list()
+		var/list/templist = typesof(/obj/item/storage/box/detectivebeacon) //we have to convert type = name to name = type, how lovely!
+		for(var/V in templist)
+			var/atom/A = V
+			detective_item_list[initial(A.name)] = A
+	return detective_item_list
+
+/obj/item/storage/box/detectivebeacon
+	name = "The Private Investigator"
+	desc = "Gritty, rude, and answers all questions in a sarcastic tone. The Private Investigator likes to rely more on his own senses and intuition, instead of this fancy new technology. Contains your standard gear, and an olfaction injector."
+
+/obj/item/storage/box/detectivebeacon/PopulateContents()
+	new /obj/item/clothing/suit/det_suit/noir(src)
+	new /obj/item/clothing/under/rank/security/detective/grey(src)
+	new /obj/item/clothing/head/fedora (src)
+	new /obj/item/ammo_box/c38(src)
+	new /obj/item/dnainjector/privatedick(src)
```

Comment:
no way you are giving him 3 roundstart free mutations. Stop.

Reply:
2 of them are bad~neutral, meant to compensate for the olfaction.

Well, at least noir is. Chav is just for the b'itish gimmick.

Reply:
noir is already in his new glasses. This is straight up bad idea.